src/Sulu/Bundle/MediaBundle/Resources/public/js/validation/types/mediaSelection.js

```diff
@@ -15,8 +15,10 @@ define([
     'use strict';
 
     var dataChangedHandler = function(data, $el) {
-        App.emit('sulu.preview.update', $el, data);
-        App.emit('sulu.content.changed');
+        if (!!data && !!$el) {
```

Comment:
Aren't you  just fixing symptoms here? To me the real issue looks like the `sulu.media-selection.images.data-changed` event is thrown twice, once even missing the data.